src/plugins/data/public/ui/filter_bar/filter_editor/range_value_input.tsx

```diff
@@ -84,6 +84,7 @@ function RangeValueInputUI(props: Props) {
   return (
     <div>
       <EuiFormControlLayoutDelimited
+        fullWidth={false}
```

Comment:
As mentioned in my other comment I don't see why this value input is going to have a different style than phrases.  I think instead you should change make this full width like you did the other input fields and change the aria-label below to just a normal label.

Reply:
Thank you so much for taking a look!

About the full-width for the range:
I did so because, In my opinion, it wasn't looking great. I added the full-width prop so for consistency.
But sure, I will make a change and update the photo. Tell me what you think after.

About the label:
sadly that's a prop from [EUI](https://elastic.github.io/eui/#/forms/form-controls#form-control-layout-delimited)